<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Spam4u</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="tabPosition">
         <enum>QTabWidget::North</enum>
        </property>
        <property name="tabShape">
         <enum>QTabWidget::Triangular</enum>
        </property>
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>Message</string>
         </attribute>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QPlainTextEdit" name="messageEdit"/>
            </item>
            <item>
             <widget class="QCommandLinkButton" name="beginSpammingButton">
              <property name="text">
               <string>Begin spamming</string>
              </property>
              <property name="autoDefault">
               <bool>false</bool>
              </property>
              <property name="description">
               <string>Begins spamming the selected eMail addresses with the message</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_2">
         <attribute name="title">
          <string>eMail account setup</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_7">
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_2">
                <item>
                 <widget class="QLabel" name="label">
                  <property name="text">
                   <string>email addresses:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="sendTimeSpinBox">
                  <property name="toolTip">
                   <string>How many times to send an email to a person.</string>
                  </property>
                  <property name="suffix">
                   <string> time(s)</string>
                  </property>
                  <property name="prefix">
                   <string/>
                  </property>
                  <property name="minimum">
                   <number>1</number>
                  </property>
                  <property name="maximum">
                   <number>256</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_6">
                <item>
                 <widget class="QListWidget" name="victimListWidget">
                  <property name="selectionMode">
                   <enum>QAbstractItemView::ExtendedSelection</enum>
                  </property>
                 </widget>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_2">
                  <item>
                   <widget class="QPushButton" name="addVictimButton">
                    <property name="text">
                     <string>Add &amp;new victim</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="editSelectedVictimButton">
                    <property name="text">
                     <string>&amp;Edit selected victim</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="removeSelectedVictimButton">
                    <property name="text">
                     <string>&amp;Remove selected victim(s)</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_5">
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_4">
                <item>
                 <spacer name="verticalSpacer_2">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeType">
                   <enum>QSizePolicy::Fixed</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>20</width>
                    <height>40</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout_3">
                  <item>
                   <widget class="QLineEdit" name="usernameEdit">
                    <property name="readOnly">
                     <bool>false</bool>
                    </property>
                    <property name="placeholderText">
                     <string>Username</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <layout class="QHBoxLayout" name="horizontalLayout">
                    <item>
                     <widget class="QLineEdit" name="smtpServerEdit">
                      <property name="placeholderText">
                       <string>SMTP server address</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QSpinBox" name="portSpinBox">
                      <property name="toolTip">
                       <string>Port</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                  <item>
                   <widget class="QLineEdit" name="passwordEdit">
                    <property name="inputMask">
                     <string/>
                    </property>
                    <property name="echoMode">
                     <enum>QLineEdit::Password</enum>
                    </property>
                    <property name="placeholderText">
                     <string>Password</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <widget class="QCommandLinkButton" name="setupMailAccountButton">
                  <property name="text">
                   <string>Setup eMail account</string>
                  </property>
                  <property name="description">
                   <string>Logs in to your eMail account to prepare for spamming</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <spacer name="verticalSpacer">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeType">
                 <enum>QSizePolicy::Fixed</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>198</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QProgressBar" name="progressBar">
          <property name="value">
           <number>0</number>
          </property>
          <property name="textVisible">
           <bool>true</bool>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="invertedAppearance">
           <bool>false</bool>
          </property>
          <property name="textDirection">
           <enum>QProgressBar::TopToBottom</enum>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionGitHub_project"/>
    <addaction name="actionGitHub_wiki"/>
    <addaction name="actionProject_page"/>
    <addaction name="separator"/>
    <addaction name="actionAbout_Spam4u"/>
   </widget>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionGitHub_project">
   <property name="icon">
    <iconset resource="assets.qrc">
     <normaloff>:/logos/assets/logos/GitHub-Mark/PNG/GitHub-Mark-32px.png</normaloff>:/logos/assets/logos/GitHub-Mark/PNG/GitHub-Mark-32px.png</iconset>
   </property>
   <property name="text">
    <string>GitHub repository</string>
   </property>
  </action>
  <action name="actionGitHub_wiki">
   <property name="icon">
    <iconset resource="assets.qrc">
     <normaloff>:/logos/assets/logos/GitHub-Mark/PNG/GitHub-Mark-32px.png</normaloff>:/logos/assets/logos/GitHub-Mark/PNG/GitHub-Mark-32px.png</iconset>
   </property>
   <property name="text">
    <string>GitHub wiki</string>
   </property>
  </action>
  <action name="actionProject_page">
   <property name="text">
    <string>Project page</string>
   </property>
  </action>
  <action name="actionAbout_Spam4u">
   <property name="text">
    <string>About Spam4u</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="assets.qrc"/>
 </resources>
 <connections/>
</ui>
